components/atom/panel/src/index.scss

```diff
@@ -32,6 +33,7 @@ $vertical-alignments-atom-panel: ('top', 'center', 'bottom') !default;
   }
 
   &-color {
+    border-radius: $bdrs-atom-panel;
```

Comment:
Creo que no deberiamos asociar el border radius al panel color, en todo caso, en la vista mobile el panel-color no tiene border radius en IJ

Reply:
If we think mobile first, the `$bdrs` should be `none` by default, and apply a media query to be rounded only for Desktop.

Reply:
Both Components, ColorPanel && ImagePannel must include its baseClassName = `sui-atom-panel`

Reply:
They do not have this baseClassName, there is a baseClassName for every single Panel (color, image) if we need to do a refactor on this maybe it gets out of the scope of this PR